<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-3PAP07V" failures="3" tests="3" name="Test" time="39.935" errors="0" timestamp="2022-12-07T17:57:05 IST">
  <testcase classname="Tests.TestCase" name="launchApp" time="5.491">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Tests.TestCase.launchApp(TestCase.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- launchApp -->
  <testcase classname="Tests.TestCase" name="launchApp1" time="3.013">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Tests.TestCase.launchApp1(TestCase.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- launchApp1 -->
  <testcase classname="Tests.TestCase" name="launchApp2" time="3.525">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Tests.TestCase.launchApp2(TestCase.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- launchApp2 -->
</testsuite> <!-- Test -->
